Kernel/Net/NetworkAdapter.cpp

```diff
@@ -95,7 +95,7 @@ void NetworkAdapter::did_receive(ReadonlyBytes payload)
         on_receive();
 }
 
-size_t NetworkAdapter::dequeue_packet(u8* buffer, size_t buffer_size, Time& packet_timestamp)
+size_t NetworkAdapter::dequeue_packet(u8* buffer, size_t buffer_size, UnixDateTime& packet_timestamp)
```

Comment:
in many places it is suggested that the type should be named `UnixTimestamp`; not sure if `UnixDateTime` has concept of calendar built in (the first question: which calendar etc.).

Reply:
The terminology "DateTime" was agreed to be most explict in our Discord discussions.

Reply:
That will apply to all types that contain date and time information; the major exception will be MonotonicTime which does not have the concept of a "date" since its starting point is unknown and a date cannot be derived.